reference/tools/tiup/mirrors.md

```diff
@@ -0,0 +1,181 @@
+      --tikv strings                Specify the versions for component tikv
+      --pd strings                  Specify the versions for component pd
+      --playground strings          Specify the versions for component playground
+      --client strings              Specify the versions for component client
+      --prometheus strings          Specify the versions for component prometheus
+      --package strings             Specify the versions for component package
+      --grafana strings             Specify the versions for component grafana
+      --alertmanager strings        Specify the versions for component alertmanager
+      --blackbox_exporter strings   Specify the versions for component blackbox_exporter
+      --node_exporter strings       Specify the versions for component node_exporter
+      --pushgateway strings         Specify the versions for component pushgateway
+      --tiflash strings             Specify the versions for component tiflash
+      --drainer strings             Specify the versions for component drainer
+      --pump strings                Specify the versions for component pump
+      --cluster strings             Specify the versions for component cluster
+      --mirrors strings             Specify the versions for component mirrors
+      --bench strings               Specify the versions for component bench
+      --insight strings             Specify the versions for component insight
+      --doc strings                 Specify the versions for component doc
+      --ctl strings                 Specify the versions for component ctl
+  -h, --help                        help for tiup
+```
+
+The basic use of the `tiup mirrors` command is as follows:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup mirrors <target-dir> [global-version] [flags]
+```
+
+- `target-dir`: The directory which stores cloned data.
+- `global-version`: Used to quickly set a global version for all components.
+
+The `tiup mirrors` command provides a number of optional flags (might be more in the future). These flags can be divided into four categories according to their purposes:
+
+- Determines whether to override local packages
+
+    The `--overwrite` flag determines whether to override the local package with the package of the official image, if the specified `<target-dir>` contains the package you need to download. If you set this flag, the local package is overridden.
+
+- Determines whether to use the full clone
+
+    If you specify the `--full` flag, you can clone the official image completely.
+
+    > **Note:**
+    >
+    > If `--full` and the other flags are not specified, only some meta information is cloned.
+
+- Determines whether to clone packages from the specific platform
+
+    If you only clone packages from one or more specific platforms, use `-os` and `-arch` to specify the platform. For example:
+
+    - Execute the `tiup mirros <target-dir> --os=linux` command to clone from linux.
```

Comment:
`to clone from linux` -> `to clone for linux`

Reply:
the `--os` and `--arch` flags specific download packages that  can run on these `os` and `arch`